### PR TITLE
Fixed Issue #894

### PR DESCRIPTION
**Pull request by AdithyaKrishnaK**
Added confetti and Try Next button

### CODE REVIEW

File: activities/Calligra.activity/index.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
 <script>if (window.module) module = window.module;</script>
 <script src="lib/require.js"></script>
 <script type="text/javascript" src="../../cordova.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/canvas-confetti@1.3.1/dist/confetti.browser.min.js"></script>
```
Comment (llaske) [inline activities/Calligra.activity/index.html:27]: Please copy the confetti file locally so the activity will be able to work offline.

File: activities/Calligra.activity/js/activity.js [hunk 2]
```diff
@@ -147,6 +147,23 @@ var app = new Vue({
...
+			var vm = this;
+			var index = -1;
+			for (var i = 0 ; i < vm.currentTemplate.images.length ; i++) {
+				if (vm.currentTemplate.images[i] == item) {
+					index = i;
+					break;
+				}
+			}
+			index++;
+			if (index === 26) {
```
Comment (llaske) [inline activities/Calligra.activity/js/activity.js:161]: 26 is not always the number of items in the template! Try with letters.

File: activities/Calligra.activity/js/player.js [hunk 3]
```diff
@@ -8,6 +8,7 @@ var Player = {
 				<canvas id="letter"></canvas>
 			</div>
 			<button id="player-restart" class="player-restart"></button>
+			<button id="player-next-letter" class="player-next" v-on:click="nextItem()">Try next</button>
```
Comment (llaske) [inline activities/Calligra.activity/js/player.js:11]: Will be better to use an icon instead. For example something like https://github.com/llaske/sugarizer/blob/dev/activities/MindMath.activity/icons/pass-icon.svg

**Review by llaske — CHANGES_REQUESTED** [review]
Nice start!

See my comments.
Plus, you should update the tutorial to explain what is this new button.

File: activities/Calligra.activity/index.html (hunk 1 above)
Comment (AdithyaKrishnaK) [reply activities/Calligra.activity/index.html:27]: While adding confetti file with npm I am getting message 2 low severity vulnerabilties found

Comment (llaske) [reply activities/Calligra.activity/index.html:27]: Include the same file than the one in Tangram activity, there is no message in Tangram activity

**Comment by AdithyaKrishnaK** [comment]
I am having trouble adding to tutorial. I don't understand where the title and description are stored and how they are accessed here https://github.com/AdithyaKrishnaK/sugarizer/blob/841bc39abf0413f146f80ae76be46aa4fa61ab32/activities/Calligra.activity/js/tutorial.js#L46

**Comment by AdithyaKrishnaK** [comment]
What am I supposed to do for other languages?

**Comment by llaske** [comment]
Good job. Thanks.

**Pull request merged**